docker/Dockerfile.tensorrt

```diff
@@ -0,0 +1,38 @@
+FROM nvcr.io/nvidia/tensorrt:23.04-py3
+
+ENV NVIDIA_DRIVER_CAPABILITIES=compute,video
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get -y install \
+    libavfilter-dev \
+    libavformat-dev \
+    libavcodec-dev \
+    libswresample-dev \
+    libavutil-dev\
+    wget \
+    build-essential \
+    ninja-build \
+    cmake \
+    git \
+    python3 \
+    python3-pip \
+    python-is-python3
+
+ARG GEN_PYTORCH_EXT=1
+# Build vpf
+RUN mkdir /src
+WORKDIR /src
+COPY src src
+COPY setup.py setup.py
+COPY CMakeLists.txt CMakeLists.txt
+RUN python3 -m pip install --no-cache-dir scikit-build \
+    && python3 -m pip install --no-cache-dir . 
```

Comment:
pip should automatically install scikit-build. if not pip/setuptools is too old.

Reply:
Aaaah ok interesting never worked for me is this a very recent version that is needed ?

Reply:
It think the problem is that the info is stored in our (cluttered) pyproject.toml which is only regarded by newer versions of setuptools
https://github.com/theHamsta/VideoProcessingFramework/blob/fdf418647dc1cf25cb1299b4066c5b3d5d1b26f9/pyproject.toml#L31-L39